### PR TITLE
Remove SSminimap

### PR DESCRIPTION
**Pull request by SpaceManiac**
Return of #37112.

@Carbonhell from Hippie reported an SSminimap crash on 512.1440 due to accidentally leaving the config on.

Nothing uses this subsystem, its maps look terrible, and it still after all this time is prone to crashes. If someone actually wants to start using these maps again they can just add the SS back in when they do that.

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

File: tools/deploy.sh [hunk 1]
```diff
@@ -6,7 +6,6 @@
 
 mkdir -p \
     $1/_maps \
-    $1/icons/minimaps \
```
Comment (Cyberboss) [inline tools/deploy.sh]: this was with -p, readd `$1/icons`

**Review by Cyberboss — DISMISSED** [review]

**Comment by Carbonhell** [comment]
tbh we're not sure that is the cause,considering the restarts started yesterday, i'll see if this happens again anyway now that we disabled it

**Comment by MrStonedOne** [comment]
The crash was fixed, if it came back we should be working with byond to fix it, as ssminimap only causes crashes by how much it uses blend(), other shit uses blend().

/tg/ has `generate_minimaps` enabled on all servers.

**Comment by MrStonedOne** [comment]
Also I think somebody, anybody, should just re-add the minimap to the crew monitor now that we have it updating.

**Comment by WJohn** [comment]
Crew monitor minimap is cool, but I wish it was used for other systems. Imagine if it were used for power monitoring consoles to show you all the APCs on the station and if any were having power issues, or for air alarms and showing you where THOSE were happening and what kind.

**Comment by vuonojenmustaturska** [comment]
Crew monitor minimap would be *cool*, but the work involved in making them and making them not shit is not small and they will go completely unused as we have a vastly superior player-finding system in-game available via pinpointers. 

Also, can you even display byond-generated minimaps ingame without crashing the clients?

**Comment by MrStonedOne** [comment]
Yes, and the crew monitor was using the minimap until somebody removed it for some reason.

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
In general a soft no from me

>Nothing uses this subsystem

The one thing that did was removed in a refactor, because at the time byond had a blend() crash bug, that crash bug was fixed, the minimap should be restored to the crew monitor, and I won't support removing this subsystem until that is attempted.

Also the maps this generates are publically available under parsed logs.

https://tgstation13.org/parsed-logs/basil/data/minimaps

image

Meaning they can be used by third party applications like statbus if needed. a functionality i'd like to preserve

**Comment by nfreader** [comment]
Statbus generates its own maps nowadays thanks to SpacemanDMM. I tried to use the minimaps briefly, but they're not as pretty as I'd like :\

**Comment by optimumtact** [comment]
@MrStonedOne are you gonna put yoru foot down over this?

**Comment by optimumtact** [comment]
@MrStonedOne @MrStonedOne @MrStonedOne

**Comment by MrStonedOne** [comment]
I dislike what this pr represents, the byond bug that was a show stopper is gone, any evidence brought forth that its back are inconclusive. The proper next steps here would be to re-add the minimaps to the crew monitor, potentially even giving them directly to the players to access as a real minimap in game, (maybe even with area based color shading to represent departments,) not removing the minimap subsystem.

But I have no actual problem with this pr itself, hence the "soft no"

So i guess my response is :confused:

**Comment by SpaceManiac** [comment]
The PR was created out of a suggestion and little more that SSminimap init might be crashing. As you've said it's enabled on the servers and hasn't been causing problems as far as anyone can tell, so I'm not really worried about it on that axis.

However, nothing uses these maps right now. They're dead code and dead init time (~10s when a map has changed last I checked). If anyone actually wants to add the minimaps back in to the game they can just revert this PR at that time.

**Pull request merged**